.github/workflows/test.yml

```diff
@@ -42,7 +42,6 @@ jobs:
       - name: Install Compose
         uses: ndeloof/install-compose-action@v0.0.1
         with:
-          version: v2.1.0 # defaults to 'latest'
```

Comment:
We shoudl have a pinned version. And not default to latest. To avoid un reproducible issues.

Reply:
I have made the update now.